cicd/42c-conf-examples/mapping-and-discovery/42c-conf.yaml

```diff
@@ -19,8 +21,10 @@
 # with its contents.
 #
 # On the other hand, the sample 'petstore.yaml' found in the discovery phase is uploaded
```

Comment:
"On the other hand, the sample 'petstore.yaml' found in the discovery phase is uploaded to the designated API collection." --> "On the other hand, the sample file 'petstore.yaml' is not mapped, so the plugin uploads to its default API collection."